pkg/operator/ceph/object/livenessprobe.go

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
nit: I think it better to add this the liveness probe changes to spec.go and spec_test.go rather than creating new files.

Reply:
Done.